scripts/check_overall_backend_test_coverage.py

```diff
@@ -0,0 +1,67 @@
+# Copyright 2022 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""This script checks if backend overall line coverage is 100%."""
+
+from __future__ import annotations
+
+import os
+import re
+import subprocess
+import sys
+
+from scripts import common
+
+
+def main() -> None:
+    """Checks if backend overall line coverage is 100%."""
+    env = os.environ.copy()
+    cmd = [
+        sys.executable, '-m', 'coverage', 'report',
+        '--omit="%s*","third_party/*","/usr/share/*"'
+        % common.OPPIA_TOOLS_DIR, '--show-missing']
+    process = subprocess.run(
+        cmd, capture_output=True, encoding='utf-8', env=env,
+        check=False)
+    if process.stdout.strip() == 'No data to report.':
+        raise RuntimeError(
+            'Run backend tests before running this script. %s' % process
+        )
+    if process.returncode:
+        raise RuntimeError(
+            'Failed to calculate coverage because subprocess failed. %s'
+            % process
```

Comment:
Will this include the stderr and stdout from the process? We'll want enough info to debug the failure

Reply:
Added them explicitely now.